spec/dummy/app/assets/javascripts/application.js

```diff
@@ -0,0 +1,13 @@
+// This is a manifest file that'll be compiled into application.js, which will include all the files
```

Comment:
Let's ignore the dummy app, as stated in https://github.com/ontohub/ontohub-models/blob/330c05fac30813a8c36987b3caddcd59e7447348/.rubocop.yml#L3